drivers/volume/portworx/schedops/schedops.go

```diff
@@ -36,11 +36,11 @@ type Driver interface {
 	// GetVolumeName returns the volume name based on the volume object received
 	GetVolumeName(v *volume.Volume) string
 	// GetServiceEndpoint returns the hostname of portworx service if it is present
-	GetServiceEndpoint() (string, error)
+	GetServiceEndpoint(string) (string, error)
 	// UpgradePortworx upgrades portworx to the given docker image and tag
-	UpgradePortworx(ociImage, ociTag, pxImage, pxTag string) error
+	UpgradePortworx(ociImage, ociTag, pxImage, pxTag, driverNamespace string) error
 	// IsPXReadyOnNode returns true if PX pod is up on that node, else returns false
-	IsPXReadyOnNode(n node.Node) bool
+	IsPXReadyOnNode(n node.Node, driverNamespace string) bool
```

Comment:
why can't you add GetVolumeDriverNamespace function here and call it whenever you need it. In this case you don't need to pass driverNamespace everywhere